util/CMakeLists.txt

```diff
@@ -210,6 +210,7 @@ if(WIN32)
 endif(WIN32)
 add_executable(cub_javasp ${JAVASP_SERVER_SOURCES})
 target_link_libraries(cub_javasp LINK_PRIVATE cubridsa)
+target_include_directories(cub_javasp PRIVATE ${EP_INCLUDES})
```

Comment:
Is this change related to this issue?

Reply:
Thanks for the check.
It was sorted out with the help of @hgryoo.